### PR TITLE
chore(docs): Update "Minimal Reproduction" instructions

### PR DESCRIPTION
**Pull request by labifrancis**
## Description

We are updating this document which is linked to an autoresponder we send to all created Zendesk tickets. This will help our users provide us with all the details we need to investigate an issue and also reduce back and forth on Zendesk.

### CODE REVIEW

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 1]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
 A reproducible test case is a great way to share a specific environment that causes a bug. Your reproducible test case is the best way to help people that want to help _you_.
 
 ## Steps to create a reproducible test case
-
-- Create a new Gatsby site with a starter, the official `gatsby-starter-minimal` starter is a great 'barebones' starting point here: `gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal`
-- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/). Only add plugins that are needed to demonstrate the problem.
+- Install Gatsby CLI if you haven’t using this [guide](https://www.gatsbyjs.com/docs/reference/gatsby-cli/).
+- Create a new Gatsby site with a starter, the official `gatsby-starter-minimal` starter is a great 'barebones' starting point here: `gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal` There are other Gatsby starters you can use [here](https://www.gatsbyjs.com/starters/)
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:19]:
```suggestion

- Create a new Gatsby site with the official `gatsby-starter-minimal` starter: `npx gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal`
```

No need to install the CLI globally, `npx` works fine, too. Also, they shouldn't use any other starters, it has to be minimal

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 2]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/) in the directory of your site (i.e bug-repro). Remember to update your `gatsby-config.js` with the plugin you installed and only add plugins that are needed to demonstrate the problem.
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:20]: I think this sentence should stay as-is, it doesn't add any additional information

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 3]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
 - Add the code needed to recreate the error you've seen.
-- Publish the code (your GitHub account is a good place to do this) and then link to it when [creating an issue](/contributing/how-to-file-an-issue/).
-
-## Online Development Environments
-
-Instead of cloning `gatsby-starter-minimal` to your local computer and working on it there, you can also use Online IDEs that set up the environment for you. Available options:
-
-- [CodeSandbox](https://githubbox.com/gatsbyjs/gatsby-starter-minimal)
-- [StackBlitz](https://githubblitz.com/gatsbyjs/gatsby-starter-minimal)
+- Build the site locally using `gatsby develop` and confirm you’re not seeing any errors.
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:22]:
```suggestion
- Verify that you're seeing the expected error(s) when running `gatsby develop`/`gatsby build`/`gatsby serve`
```

Well, they should see an error, that's the whole point of the reproduction. It also depends on what behavior they want to prove, so only `gatsby develop` isn't enough

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 4]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+- Publish the code (your GitHub account is a good place to do this)
+- Use the repo to build a site on Gatsby cloud and look out for any build errors.
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:24]: I think this is in the incorrect place here, for OSS issue reports we don't need Gatsby Cloud builds

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 5]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+- Share the link to the repo and build logs when reaching out to support. You can also share the repo if you’re [creating an issue](https://www.gatsbyjs.com/contributing/how-to-file-an-issue/)
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:25]:
```suggestion
- Link the public repository when [creating an issue](/contributing/how-to-file-an-issue/) or reaching out to Gatsby Cloud support
```

You need to use relative links for internal links, like it was before

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 6]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+- Install Gatsby CLI if you haven’t using this [guide](https://www.gatsbyjs.com/docs/reference/gatsby-cli/).
+- Create a new Gatsby site with a starter, the official `gatsby-starter-minimal` starter is a great 'barebones' starting point here: `gatsby new bug-repro https://github.com/gatsbyjs/gatsby-starter-minimal` There are other Gatsby starters you can use [here](https://www.gatsbyjs.com/starters/)
+- Add any Gatsby plugins that relate to the issue. For example, if you're having problems with Gatsby MDX you should install and configure [`gatsby-plugin-mdx`](/plugins/gatsby-plugin-mdx/) in the directory of your site (i.e bug-repro). Remember to update your `gatsby-config.js` with the plugin you installed and only add plugins that are needed to demonstrate the problem.
 - Add the code needed to recreate the error you've seen.
-- Publish the code (your GitHub account is a good place to do this) and then link to it when [creating an issue](/contributing/how-to-file-an-issue/).
-
-## Online Development Environments
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:24]: Why did you remove this?

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 7]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+- Share the link to the repo and build logs when reaching out to support. You can also share the repo if you’re [creating an issue](https://www.gatsbyjs.com/contributing/how-to-file-an-issue/)
 
 ## Benefits of reproducible test cases
 
 - Smaller surface area: By removing everything but the error, you don't have to dig to find the bug.
 - No need to publish secret code: You might not be able to publish your main site (for many reasons). Remaking a small part of it as a reproducible test case allows you to publicly demonstrate a problem without exposing any secret code.
 - Proof of the bug: Sometimes a bug is caused by some combination of settings on your machine. A reproducible test case allows contributors to pull down your build and test it on their machines as well. This helps verify and narrow down the cause of a problem.
 - Get help with fixing your bug: If someone else can reproduce your problem, they often have a good chance of fixing the problem. It's almost impossible to fix a bug without first being able to reproduce it.
+
+## **What if I’m unable to create a reproducible test case?**
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:34]: There's no need for making the headings bold

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 8]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+
+For Gatsby Cloud users, If you can’t create a reproducible test case, you can grant us access to you site by giving us repo permissions to reproduce the issue, creating a video of the issue preferably with [loom](https://www.loom.com/). if it’s an issue with your CMS, you can share the credentials( or create an account for us using support@gatsbyjs.com) and the reproduction steps.
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:36]: I don't see how that answers the above question. If they can't reproduce the issue in the repository, why would giving us access to it make it magically work? We should give them tips on how to make the GitHub repository work as a reproduction, not make us do the busy work.

I'm also vary of recommending people sending credentials via email, bad actors could start phishing people as then our users would know that we'd do that.

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 9]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+
+### **For Build issues:**
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:38]: This is too specific to Gatsby Cloud (without specifying it)

I think this and the hosting issues heading should be under its own "Gatsby Cloud" heading

File: docs/contributing/how-to-make-a-reproducible-test-case.md [hunk 10]
```diff
@@ -15,22 +15,48 @@ The [most important part of any bug report](https://developer.mozilla.org/en-US/
...
+4. Link to the repo for the reproducible test case
+
+### **For Hosting issues:**
+
+1. Share the path to the problematic URL
+2. What is the expected behavior of this URL?
+3. What are you experiencing on your end?
+4. Screenshot of the error message and screenshot of browser console
+
+### **For CMS Issues:**
```
Comment (LekoArts) [inline docs/contributing/how-to-make-a-reproducible-test-case.md:52]: I think this should be (in a generalized form) under the "Steps to create a reproducible test case" heading

**Review by LekoArts — CHANGES_REQUESTED** [review]

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 6 above)
Comment (labifrancis) [reply docs/contributing/how-to-make-a-reproducible-test-case.md:24]: We've always encouraged customers to reproduce issues locally and I thought this may not be needed.

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 8 above)
Comment (labifrancis) [reply docs/contributing/how-to-make-a-reproducible-test-case.md:36]: Should we remove this then? Sometimes it gets to the point where we ask for these permissions but I understand your context.

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 6 above)
Comment (LekoArts) [reply docs/contributing/how-to-make-a-reproducible-test-case.md:24]: This way they can reproduce it locally, it's just running inside a VM, not on their local computer so we're fine with those online IDEs

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 8 above)
Comment (LekoArts) [reply docs/contributing/how-to-make-a-reproducible-test-case.md:36]: Yes, if we do the comment I left under "CMS issues" I think we can remove this section for now

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 2 above)
Comment (labifrancis) [reply docs/contributing/how-to-make-a-reproducible-test-case.md:20]: Please which sentence in particular?

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 4 above)
Comment (labifrancis) [reply docs/contributing/how-to-make-a-reproducible-test-case.md]: Updated

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 7 above)
Comment (labifrancis) [reply docs/contributing/how-to-make-a-reproducible-test-case.md]: Removed the heading

File: docs/contributing/how-to-make-a-reproducible-test-case.md (hunk 10 above)
Comment (labifrancis) [reply docs/contributing/how-to-make-a-reproducible-test-case.md:55]: Done

**Review by LekoArts — APPROVED** [review]
lgtm, if you want to merge it as-is, feel free to do so

**Comment by labifrancis** [comment]
Thank you @LekoArts
